server/indexer/index.go

```diff
@@ -0,0 +1,17 @@
+package indexer
+
+type IndexRoot struct {
+	ResultDataUrl string
+	IdProperty    string
+	Indexes       map[string]interface{}
+}
+
+type EnumIndex struct {
+	Kind   string
+	Values map[string]*EnumValue
```

Comment:
```suggestion
	Kind   string                `json:"kind"`
	Values map[string]*EnumValue `json:"values"`
```

All struct fields that can be serialized or deserialized as JSON should have a `json` tag. CamelCase with the first letter capitalized is not very common in JSON.